client/src/pages/Restaurant/RestaurantMap/Map.js

```diff
@@ -0,0 +1,44 @@
+/*global google*/
+import React from "react";
+import { GoogleMap } from "@react-google-maps/api";
+
+const containerStyle = {
+  width: "100%",
+  height: "80vh",
+};
+
+function Map({latitude, longitude}) {
+  const [map, setMap] = React.useState(null);
+
+  const mapOptions = {
+    zoomControlOptions: {
+      position: google.maps.ControlPosition.RIGHT_TOP,
+    },
+    streetViewControlOptions: {
```

Comment:
See if you can find a way to remove the street view icon. No rush though, you can do this later